rock64_install_to_emmc.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+set -eo pipefail
+
+if [[ "$(id -u)" -ne "0" ]]; then
+	echo "This script requires root."
+	exit 1
+fi
+
+echo "Rock64 Release Installer!"
+echo "(C) 2017. Kamil Trzciński (https://ayufan.eu)."
+echo ""
+
+usage() {
+    echo "Usage:"
+    echo "$ $0 <system> [version]"
+    echo ""
+    echo "Systems:"
+    echo " - xenial-minimal (https://github.com/ayufan-rock64/linux-build/releases)"
+    echo " - xenial-mate (https://github.com/ayufan-rock64/linux-build/releases)"
+    echo " - xenial-i3 (https://github.com/ayufan-rock64/linux-build/releases)"
+    echo ""
+    echo "Version:"
+    echo " - latest will be used if version is not defined"
+    exit 1
+}
+
+if [[ $# -ne 1 ]] && [[ $# -ne 2 ]]; then
+    usage
+fi
+
+if [[ ! -d /sys/devices/soc.0/1c10000.sdmmc/mmc_host/mmc1 ]]; then
+    echo "You should boot from SD card"
+    exit 1
+fi
+
+if [[ ! -e /dev/mmcblk1 ]]; then
+    echo "You should boot from SD card"
+    exit 1
+fi
+
+case "$1" in
+    xenial-minimal|xenial-mate|xenial-i3)
```

Comment:
Why only these three? Why not also the other currently offered builds? i.e. armhf build, the stretch build, and the jessie-minimal and jessie-omv builds?